paasta_tools/deployd/common.py

```diff
@@ -50,7 +80,28 @@ def put(self, item, *args, **kwargs):
         Queue.put(self, item, *args, **kwargs)
 
 
-def rate_limit_instances(instances, number_per_minute, watcher_name):
+class PaastaPriorityQueue(PriorityQueue):
+
+    def __init__(self, name, *args, **kwargs):
+        self.name = name
+        PriorityQueue.__init__(self, *args, **kwargs)
```

Comment:
I didn't know that was something they'd sorted, I'll poke around and change anything old style to use `super()` 👍